cloudflare_ddns

```diff
@@ -10,7 +10,7 @@ fi
 
 # script settings
 LOGFILE=$(basename $0).log
-THROTTLE_SECONDS=300  # terminate prematurely if run too often
+THROTTLE_SECONDS=0  # terminate prematurely if run too often
```

Comment:
This appears to be a mistaken holdover from testing. Please revert or explain.

Reply:
yes, that was a holdover from testing.  I noticed right after I submitted, but I am new to contributing on git and couldn't figure out how to change.